### DISCUSSION TITLE
External Dependencies do not persist

### DISCUSSION THREAD

**Discussion by gaberger** [mikub/titanoboa#26] [General]
When I restart my server I have to redo all the external dependencies

**Comment by mikub** [comment]
You need to specify the file path as system property `boa.server.dependencies.path` e.g. in `start.sh` just add `-Dboa.server.dependencies.path=<path to the file>`

When this is not set, Titanoboa just uses default ext-dependencies file it has on classpath. 
The original reason behind this was to have the ability to pre-package some dependencies with T-boa (form times it was just an uberjar), but I have already got couple of these complains, so maybe this is not the most user-friendly approach.

One solution would be to ship extracted ext-depedencies file and already updated start scripts.
